### PR TITLE
Create credentials

### LINKED ISSUE #9: Create credentials
Ask user to enter data.

### PR DESCRIPTION
**Pull request by opixelum**
- Define credentials type
- Create main menu
- Create credentials by asking user
- No need to specify path for includes

### CODE REVIEW

File: lib/ui.c [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#include "ui.h"
+
+void mainMenu(unsigned char *isRunning)
+{
+    printf("\nPassword Manager\n\n\
+    1. New credentials\n\
+    3. Update credentials\n\
```
Comment (Invorom) [inline lib/ui.c]: 2 & 3

File: lib/ui.c [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    scanf("%s", username);
+
+    // Get email address
+    printf("    Enter the email address (optional): ");
+    char email[100];
+    scanf("%s", email);
+
+    // Ask if user wants to generate a password
+    printf("    Do you want to generate a password? (y/n): ");
+    char generatePswd[100];
```
Comment (Invorom) [inline lib/ui.c]: Variable name
Use a "choice" variable for all choices

**Review by Invorom — CHANGES_REQUESTED** [review]

File: lib/ui.c (hunk 1 above)
Comment (opixelum) [reply lib/ui.c]: Done

File: lib/ui.c (hunk 2 above)
Comment (opixelum) [reply lib/ui.c]: Done but not really optimized. Will create an issue for utils functions (get input properly, ...).

**Review by Invorom — APPROVED** [review]
Need to optimize the code

**Pull request merged**